connectors/sources/gcs.py

```diff
@@ -325,13 +331,16 @@ async def get_content(self, blob, timestamp=None, doit=None):
                 )
             )
             await async_buffer.seek(0)
```

Comment:
why do you need to keep that seek ? I think you can remove it

Reply:
Sure, thanks.